src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/yflows/YFlowCreateSpec.groovy

```diff
@@ -395,6 +395,11 @@ source: switchId="${flow.sharedEndpoint.switchId}" port=${flow.sharedEndpoint.po
 
         cleanup:
         yFlowResponse && !exc && yFlowHelper.deleteYFlow(yFlowResponse.YFlowId)
+        Wrappers.wait(WAIT_OFFSET) {
```

Comment:
there is no need in this wrapper, cause yFlowHelper.deleteYFlow contains assertion on the flow history, which is filing the delete action (the flow history logs it once it verifies the subflows are removed)

Reply:
This is workaround for another problem. One of subflows hangs 'In Progress' for a while after test is over, and then it is also removed, but this can break the next test (e.g. 'consume' some bandwidth).

Reply:
deleteYFlow cant return you true until the both subflows will be deleted (with particular status assigned), and there no need to make it once again

Reply:
deleteYFlow should be called in this test only if test is failed and exception was not raised on flow creation.